JSDOMParser.js

```diff
@@ -1118,7 +1125,7 @@
       // If this isn't a void Element, read its child nodes
       if (!closed) {
         this.readChildren(node);
-        var closingTag = "</" + localName + ">";
+        var closingTag = "</" + node._matchingTag + ">";
```

Comment:
I don't think this will match closing tags such as `</p   >` which is valid according to https://www.w3.org/TR/html5/syntax.html#end-tags

Reply:
XMLSerializer never produces those, though.

Reply:
Okay thanks for the clarification.

Reply:
This was a pre-existing issue, too, so I'll just merge this as-is, but @msujaws let me know if I should file an issue to change this to be defensive in case XMLSerializer ever changes.

Reply:
Meh, I'm fine with not worrying about this change. I was more concerned if this was unsanitized markup.